### ISSUE TITLE
Display signed publisher information

### ISSUE THREAD

**Issue by onovotny** [NuGet/NuGetGallery#6882]
The NuGet website does not currently show the publisher information from the certificate.

I would like it to show the subject name next to all listings (short search results, details page, etc). Clicking it could show the details like it does on GitHub:

![image](https://user-images.githubusercontent.com/1427284/52522670-74ce5680-2c56-11e9-838f-67592c356a06.png)

**Comment by anangaur** [comment]
Note to self: we would need to somehow marry together the owner vs. author issue and signed author information together.

**Comment by jeffkeen** [comment]
Yes! Please help me because the phone that I submitted work from is broke. My screen doesn't work but it hopefully still has my IP address and DNA. I know that I had used GitHub to go from making an app, to a Firewall then implemented Source Codes to change our technology through the Open Source Project. 
I was having extreme difficulties during the process of someone around me who was trying to inquire a hack to gain access to everything I was doing. I don't care about hacking and this was my fight to gain a healthier technology to give people a secure peace of mind for a stronger sustainable security. I need your assistance please!!!
Help me gain trust with you and ensure that I still have a place on your team. Thank you
